exonum-java-binding-core/pom.xml

```diff
@@ -144,7 +144,10 @@
           <excludes>
             <exclude>**/*IntegrationTest.java</exclude>
           </excludes>
-          <argLine>${java.vm.assertionFlag}</argLine>
+          <argLine>
+            ${argLine}
+            ${java.vm.assertionFlag}
+          </argLine>
```

Comment:
Shall this property be set for failsafe running ITs :arrow_down: ?

Reply:
Good catch, I'll look into that.

Reply:
^